### ISSUE TITLE
[EPIC] My HealtheVet on VA.gov accessibility conformance audit

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#138596]
## Description
Conduct a WCAG 2.2 Level A and AA conformance audit of My HealtheVet on VA.gov.

Task tickets for each task flow are linked to this epic.

## Supporting resources

- [Audit ticket template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?template=mhv-a11y-audit.yaml) — for managing an accessibility audit
- [Accessibility defect ticket template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?template=mhv-a11y-issue.yaml) — for reporting accessibility issues found in an accessibility audit

### Workstream 1: Pilot a task-focused conformance audit process and output

- [x] Determine which task to use for the pilot ([Slack request for candidates](https://dsva.slack.com/archives/C05QG2H510T/p1775681573212259)) — starting with appointment scheduling flow on desktop and mobile web
- [x] Create an [MHV Accessibility Conformance Audit Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?template=mhv-a11y-audit.yaml) for the task — see #139235 
- [ ] Work through the flow following the steps defined in the Audit Ticket
- [ ] Document process in this Epic ticket for future task audits

### Workstream 2: Create supporting resources and artifacts

- [ ] Create test plan worksheet
- [ ] Set up audit worksheet to support portal-wide audit of My HealtheVet on VA.gov.
- [ ] Set up issue tracking worksheet to track status of issues
- [ ] Draft proposal for issue severity rating rubric
